pontoon/base/static/js/main.js

```diff
@@ -143,6 +143,38 @@ $(function () {
         ga('send', 'event', 'ajax', 'request', settings.url);
     });
 
+    /*
+     * Display Pontoon Add-On promotion, if:
+     *
+     * - Pontoon Add-On not installed
+     * - Page loaded on Firefox or Chrome (add-on not available for other browsers)
+     */
+    var isFirefox = navigator.userAgent.indexOf('Firefox') !== -1;
+    var isChrome = navigator.userAgent.indexOf('Chrome') !== -1;
+    var downloadLink = $('#addon-promotion').find('.get');
+    setTimeout(function () {
+        if (!window.PontoonAddon || !window.PontoonAddon.installed) {
+            if (!isFirefox && !isChrome) {
+                return;
+            }
+            if (isFirefox) {
+                downloadLink.attr(
+                    'href',
+                    'https://addons.mozilla.org/sl/firefox/addon/pontoon-tools/',
```

Comment:
Should be locale independent link `https://addons.mozilla.org/firefox/addon/pontoon-tools/`.

Reply:
Ugh, nice catch!